cookbook/security/ldap.rst

```diff
@@ -0,0 +1,269 @@
+.. index::
+   single: Security; Authenticating against an LDAP server
+
+Authenticating against an LDAP server
+=====================================
+
+Symfony provides different means to work with an LDAP server.
+
+The Security component offers:
+
+* The ``ldap`` user provider, using the
+  :class:`Symfony\\Component\\Security\\Core\\User\\LdapUserProvider``
```

Comment:
Minor syntax issue: double backticks at the end of this line should be single backtick.